.circleci/config.yml

```diff
@@ -13,9 +13,9 @@ jobs:
 
   chart-verifier:
     docker:
-      - image: docker.mirror.hashicorp.services/cimg/go:1.16
+      - image: docker.mirror.hashicorp.services/cimg/go:1.19.2
     environment:
-      BATS_VERSION: "1.3.0"
+      BATS_VERSION: "1.8.2"
```

Comment:
We could remove all the circleci jobs from this file except for `update-helm-charts-index`. I think that's the only one left running in circle.

Reply:
Oh good point, I didn't notice they've already been removed from the workflows section of the file anyway so they were already unused.